css/css-fonts-3/font-variant-03.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+<title>CSS Test:  font-variant: normal; low level equivalence</title>
+<link rel="author" title="Chris Lilley" href="chris@w3.org">
+<link rel="help" href="https://www.w3.org/TR/css-fonts-3/#font-variant-prop">
+<link rel="help" href="https://www.w3.org/TR/css-fonts-3/#font-feature-settings-prop">
+<link rel="match" href="font-variant-03-ref.html">
+<meta name="assert" content="The value ‘normal’ resets all subproperties of ‘font-variant’ to their inital value. It does not reset the values of either ‘font-language-override’ or ‘font-feature-settings’">
+<style>
+  @font-face {
+    font-family: fwf;
+    src: url(support/fonts/FontWithFeatures2.otf);
```

Comment:
Are you sure this is the right font? https://github.com/w3c/web-platform-tests/pull/5875/files uses a different one.